frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_user_view.dart

```diff
@@ -160,50 +182,45 @@ class _CurrentIcon extends StatelessWidget {
       Navigator.of(context).pop();
     }
 
-    return Material(
-      color: Colors.transparent,
-      child: GestureDetector(
-        onTap: () {
-          showDialog(
-            context: context,
-            builder: (BuildContext context) {
-              return SimpleDialog(
-                title: FlowyText.medium(
-                  LocaleKeys.settings_user_selectAnIcon.tr(),
-                  fontSize: FontSizes.s16,
-                ),
-                children: <Widget>[
-                  SizedBox(
-                    height: 300,
-                    width: 300,
-                    child: IconGallery(setIcon),
-                  )
-                ],
-              );
-            },
-          );
-        },
-        child: Column(
-          children: <Widget>[
-            Align(
-              alignment: Alignment.topLeft,
-              child: Text(
-                LocaleKeys.settings_user_icon.tr(),
-                style: const TextStyle(color: Colors.grey),
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          LocaleKeys.settings_user_icon.tr(),
+          style: Theme.of(context).textTheme.titleSmall!.copyWith(
+                fontWeight: FontWeight.w500,
+                fontSize: 13,
               ),
-            ),
-            Align(
-              alignment: Alignment.centerLeft,
-              child: Container(
-                margin: const EdgeInsets.all(5.0),
-                decoration:
-                    BoxDecoration(border: Border.all(color: Colors.grey)),
-                child: svgWidget('emoji/$iconUrl', size: const Size(60, 60)),
-              ),
-            ),
-          ],
         ),
-      ),
+        InkWell(
+          borderRadius: Corners.s6Border,
+          hoverColor: Theme.of(context).colorScheme.secondaryContainer,
+          onTap: () {
+            showDialog(
+              context: context,
+              builder: (BuildContext context) {
+                return SimpleDialog(
+                  title: FlowyText.medium(
+                    LocaleKeys.settings_user_selectAnIcon.tr(),
+                    fontSize: FontSizes.s16,
+                  ),
+                  children: <Widget>[
+                    SizedBox(
```

Comment:
Is this going to be ok if we resize the window to be really small?

Reply:
I think we have set some minimum sizes for the window. I just tried and it looks fine.
![image](https://user-images.githubusercontent.com/14248245/235534164-42e5cd3b-c4c5-435c-8442-56596bc8125d.png)